src/main/kotlin/uk/gov/dwp/dataworks/model/Model.kt

```diff
@@ -9,5 +9,6 @@ data class Model @JsonCreator constructor(
         val albName: String,
         val containerPort: Int = 443,
         val jupyterCpu: Int = 512,
-        val jupyterMemory: Int = 512
+        val jupyterMemory: Int = 512,
+        val listOfAdditionalPermissions: List<String> = emptyList()
```

Comment:
probably better calling this `additionalPermissions`, no need to qualify it as a list